test/metabase/query_processor_test/alternative_date_test.clj

```diff
@@ -178,7 +178,9 @@
 (deftest iso-8601-text-fields
   (testing "text fields with semantic_type :type/ISO8601DateTimeString"
     (testing "return as dates"
-      (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :sqlite :oracle :sparksql)
+      (mt/test-drivers (-> (sql-jdbc.tu/sql-jdbc-drivers)
+                           (conj :bigquery-cloud-sdk)
+                           (disj :sqlite :oracle :sparksql))
```

Comment:
Can you add a comment explaining the selection of drivers?

Reply:
I honestly don't know about the excluded drivers, the reason we have bigquery there is that we could reuse the tests for it...

Reply:
So the mystery remains...